recipes/py-trees/meta.yaml

```
@@ -0,0 +1,77 @@
+{% set name = "py-trees" %}
+{% set version = "2.2.3" %}
+{% set python_min = "3.8" %}
+{% set python_max = "4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/py_trees-{{ version }}.tar.gz
+  sha256: 8a2024ca6cb966c3c571d91e2d60c90cc562c4d237a0af8ab7a7c27c2cff0795
+
+build:
+  entry_points:
+    - py-trees-render = py_trees.programs.render:main
+    - py-trees-demo-action-behaviour = py_trees.demos.action:main
+    - py-trees-demo-behaviour-lifecycle = py_trees.demos.lifecycle:main
+    - py-trees-demo-blackboard = py_trees.demos.blackboard:main
+    - py-trees-demo-blackboard-namespaces = py_trees.demos.blackboard_namespaces:main
+    - py-trees-demo-blackboard-remappings = py_trees.demos.blackboard_remappings:main
+    - py-trees-demo-context-switching = py_trees.demos.context_switching:main
+    - py-trees-demo-display-modes = py_trees.demos.display_modes:main
+    - py-trees-demo-dot-graphs = py_trees.demos.dot_graphs:main
+    - py-trees-demo-either-or = py_trees.demos.either_or:main
+    - py-trees-demo-eternal-guard = py_trees.demos.eternal_guard:main
+    - py-trees-demo-logging = py_trees.demos.logging:main
+    - py-trees-demo-pick-up-where-you-left-off = py_trees.demos.pick_up_where_you_left_off:main
+    - py-trees-demo-selector = py_trees.demos.selector:main
+    - py-trees-demo-sequence = py_trees.demos.sequence:main
+    - py-trees-demo-tree-stewardship = py_trees.demos.stewardship:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >={{ python_min }},<{{ python_max }}
```

```suggestion
    - python {{ python_min }}
```

`host` should only pin against the min python version